### ISSUE TITLE
[Bug][SfCartesianChart][Web]   OnChartTouchInteractionUp is only triggered on press

### ISSUE THREAD

**Issue by monolidth** [syncfusion/flutter-widgets#867]
Hello, thanks for the excellent chart library. 
I face a problem regarding the onChartTouchInteractionUp listener. The listener is only triggered via a mouse click and not if the interaction is up. Is there any workaround fix for this? 

This problem appears on the web only.

Regards,

William

**Comment by SriramKiranSenthilkumar** [comment]
Hi @monolidth,

Greetings from Syncfusion.

We have checked your query at our end, and we would like to let you know that the behavior of the onChartInteractionDown, onChartInteractionUp and onChartInteractionMove callbacks is that,

The [onChartInteractionDown](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/SfCircularChart/onChartTouchInteractionDown.html) callback fires when the finger is touched down on the chart area and in the case of mouse interaction, it will fire when the mouse left click button is pressed down.

The [onChartInteractionUp](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/SfCartesianChart/onChartTouchInteractionUp.html) callback fires when the finger is tapped or clicked on the chart area i.e., the finger has stopped making contact with the chart area and in the case of mouse interaction, it will fire when the mouse left click button is released after pressing down that is on single mouse click.

The [onChartInteractionMove](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/SfCartesianChart/onChartTouchInteractionMove.html) callback fires when the finger is touched down and moved on the chart area and in the case of mouse interaction, it will fire when the mouse button is pressed down and moved on the chart area without releasing the button.

We have ensured the [onChartInteractionUp](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/SfCartesianChart/onChartTouchInteractionUp.html) callback is working fine in mouse up properly. As we are not sure about your exact requirement, we request you to share us with more information regarding your requirement in detail so that it will help us assist you in a better way.



Regards,

Sriram Kiran
